.github/workflows/vectors-matrix.yml

```
@@ -0,0 +1,68 @@
+name: Canonicalization Vectors (Python/Rust/Go)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)